src/components/materialization/Create/index.tsx

```diff
@@ -127,51 +128,53 @@ function MaterializationCreate() {
                     'https://docs.estuary.dev/guides/create-dataflow/#create-a-materialization',
             }}
         >
-            <EndpointConfigHydrator>
-                <ResourceConfigHydrator>
-                    <EntityCreate
-                        title="browserTitle.materializationCreate"
-                        entityType={entityType}
-                        draftSpecMetadata={draftSpecsMetadata}
-                        resetState={resetState}
-                        errorSummary={
-                            <ValidationErrorSummary
-                                errorsExist={detailsFormErrorsExist}
-                            />
-                        }
-                        toolbar={
-                            <EntityToolbar
-                                GenerateButton={
-                                    <MaterializeGenerateButton
-                                        disabled={!hasConnectors}
-                                        callFailed={helpers.callFailed}
-                                    />
-                                }
-                                TestButton={
-                                    <EntityTestButton
-                                        disabled={!hasConnectors}
-                                        callFailed={helpers.callFailed}
-                                        closeLogs={handlers.closeLogs}
-                                        logEvent={
-                                            CustomEvents.MATERIALIZATION_TEST
-                                        }
-                                    />
-                                }
-                                SaveButton={
-                                    <EntitySaveButton
-                                        disabled={!draftId}
-                                        callFailed={helpers.callFailed}
-                                        closeLogs={handlers.closeLogs}
-                                        logEvent={
-                                            CustomEvents.MATERIALIZATION_CREATE
-                                        }
-                                    />
-                                }
-                            />
-                        }
-                    />
-                </ResourceConfigHydrator>
-            </EndpointConfigHydrator>
+            <DetailsFormHydrator>
```

Comment:
Addition of a details form store hydrator. No additional changes were made to this file.